swarm/tools/runs_gc.py

```
@@ -74,15 +75,24 @@ def age_days(self) -> float:
-    try:
-        for entry in path.rglob("*"):
-            if entry.is_file():
-                try:
-                    total += entry.stat().st_size
-                except OSError:
-                    pass
-    except OSError:
-        pass
+
+    # PERFORMANCE OPTIMIZATION (Bolt): Replace rglob with os.scandir to speed up directory traversal
+    def _calc_size(p: str):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_calc_size`

Add return type annotation: `None`

(ANN202)